python/tvm/micro/model_library_format.py

```diff
@@ -156,10 +170,11 @@ def export_model_library_format(mod: graph_executor_factory.GraphExecutorFactory
     with open(tempdir.relpath("relay.txt"), "w") as f:
         f.write(str(mod.ir_mod))
 
-    graph_config_dir_path = tempdir.relpath(os.path.join("runtime-config", "graph"))
-    os.makedirs(graph_config_dir_path)
-    with open(os.path.join(graph_config_dir_path, "graph.json"), "w") as f:
-        f.write(mod.graph_json)
+    if not is_aot:
```

Comment:
Well the point is that I don't need any configuration with AOT. This is only saying "don't store the json if it is using the aot executor"

Reply:
I think we should just use the factored out function which determines the runtime and moreover I think its better if we encapsulate into a function whether  "mod" requires config data and if so what that would be. e.g., if "graph" it would be the json, etc. Thus it would be extensible when configs needed to be added based on the executor. @areusch @giuseros  WDYT?

Reply:
@manupa-arm just to clarify, do you mean factor out this logic into a graph-specific function and invoke based on the executor type? if so I agree w/ that. I'm not convinced AOT will always require 0 configuration, particularly given conversation in https://discuss.tvm.apache.org/t/mapping-tensorir-te-to-heterogenous-systems/9617